### PR TITLE
Fixes AB#1741220 - update proxy info

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

Fixes [AB#1741220](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1741220) - update proxy info

## PR Context

Select the area of the Table of Contents containing the documents being changed.

**Conceptual content**
- [ ] Overview and Install
- [ ] Learning PowerShell
  - [ ] PowerShell 101
  - [ ] Deep dives
  - [ ] Remoting
- [ ] Release notes (What's New)
- [ ] Windows PowerShell
  - WMF, ISE, release notes, etc.
- [ ] DSC articles
- [ ] Community resources
- [ ] Sample scripts
- [ ] Gallery articles
- [ ] Scripting and development
  - [ ] Legacy SDK

**Cmdlet reference & about_ topics**
- [x] Version 7.1 preview content
- [x] Version 7.0 content
- [ ] Version 6 content
- [ ] Version 5.1 content

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: reference/7.0/Microsoft.PowerShell.Utility/Invoke-RestMethod.md [hunk 1]
```diff
@@ -1229,32 +1229,27 @@ strings.
 
 Some features may not be available on all platforms.
 
-For more information about how .NET provides proxy services to PowerShell, see
-[Accessing the Internet Through a Proxy](/dotnet/framework/network-programming/accessing-the-internet-through-a-proxy).
+Because of changes in .NET Core 3.1, PowerShell 7.0 and higher use the
+[HttpClient.DefaultProxy](/dotnet/api/system.net.http.httpclient.defaultproxy?view=netcore-3.1)
+Property to determine the proxy configuration.
```
Comment (chasewilson) [inline reference/7.0/Microsoft.PowerShell.Utility/Invoke-RestMethod.md:1234]:
```suggestion
property to determine the proxy configuration.
```

File: reference/7.0/Microsoft.PowerShell.Utility/Invoke-RestMethod.md [hunk 2]
```diff
@@ -1229,32 +1229,27 @@ strings.
...
-Because of changes in .NET Core 3.1, PowerShell 7.0 and higher supports proxy configuration defined
-by the following environment variables:
+The value of this property is different rules depending on your platform:
 
-- `$env:http_proxy` or `$env:HTTP_PROXY`
+- **For Windows**: Reads proxy configuration from environment variables or, if those are not
+  defined, from the user's proxy settings.
+- **For macOS**: Reads proxy configuration from environment variables or, if those are not defined,
+  from the system's proxy settings.
+- **For Linux**: Reads proxy configuration from environment variables or, in case those are not
```
Comment (chasewilson) [inline reference/7.0/Microsoft.PowerShell.Utility/Invoke-RestMethod.md:1242]:
```suggestion
- **For Linux**: Reads proxy configuration from environment variables or, if those are not
```

It just reads better if they are consistent to me. Feel free to ignore

File: reference/7.0/Microsoft.PowerShell.Utility/Invoke-WebRequest.md [hunk 3]
```diff
@@ -1188,29 +1188,27 @@ Beginning with PowerShell 6.0.0 `Invoke-WebRequest` supports basic parsing only.
 For more information, see
 [BasicHtmlWebResponseObject](/dotnet/api/microsoft.powershell.commands.basichtmlwebresponseobject).
 
-Because of changes in .NET Core 3.1, PowerShell 7.0 and higher supports proxy configuration defined
-by the following environment variables:
+Because of changes in .NET Core 3.1, PowerShell 7.0 and higher use the
+[HttpClient.DefaultProxy](/dotnet/api/system.net.http.httpclient.defaultproxy?view=netcore-3.1)
+Property to determine the proxy configuration.
```
Comment (chasewilson) [inline reference/7.0/Microsoft.PowerShell.Utility/Invoke-WebRequest.md:1193]:
```suggestion
property to determine the proxy configuration.
```

File: reference/7.0/Microsoft.PowerShell.Utility/Invoke-WebRequest.md [hunk 4]
```diff
@@ -1188,29 +1188,27 @@ Beginning with PowerShell 6.0.0 `Invoke-WebRequest` supports basic parsing only.
...
 
-- `$env:http_proxy` or `$env:HTTP_PROXY`
+The value of this property is different rules depending on your platform:
 
-  Set this variable to the endpoint of an HTTP proxy. For example: `http://127.0.0.1:3001`
+- **For Windows**: Reads proxy configuration from environment variables or, if those are not
+  defined, from the user's proxy settings.
+- **For macOS**: Reads proxy configuration from environment variables or, if those are not defined,
+  from the system's proxy settings.
+- **For Linux**: Reads proxy configuration from environment variables or, in case those are not
```
Comment (chasewilson) [inline reference/7.0/Microsoft.PowerShell.Utility/Invoke-WebRequest.md:1201]:
```suggestion
- **For Linux**: Reads proxy configuration from environment variables or, if those are not
```

File: reference/7.1/Microsoft.PowerShell.Utility/Invoke-RestMethod.md [hunk 5]
```diff
@@ -1230,32 +1230,27 @@ strings.
 
 Some features may not be available on all platforms.
 
-For more information about how .NET provides proxy services to PowerShell, see
-[Accessing the Internet Through a Proxy](/dotnet/framework/network-programming/accessing-the-internet-through-a-proxy).
+Because of changes in .NET Core 3.1, PowerShell 7.0 and higher use the
+[HttpClient.DefaultProxy](/dotnet/api/system.net.http.httpclient.defaultproxy?view=netcore-3.1)
+Property to determine the proxy configuration.
```
Comment (chasewilson) [inline reference/7.1/Microsoft.PowerShell.Utility/Invoke-RestMethod.md:1235]:
```suggestion
property to determine the proxy configuration.
```

File: reference/7.1/Microsoft.PowerShell.Utility/Invoke-RestMethod.md [hunk 6]
```diff
@@ -1230,32 +1230,27 @@ strings.
...
-Because of changes in .NET Core 3.1, PowerShell 7.0 and higher supports proxy configuration defined
-by the following environment variables:
+The value of this property is different rules depending on your platform:
 
-- `$env:http_proxy` or `$env:HTTP_PROXY`
+- **For Windows**: Reads proxy configuration from environment variables or, if those are not
+  defined, from the user's proxy settings.
+- **For macOS**: Reads proxy configuration from environment variables or, if those are not defined,
+  from the system's proxy settings.
+- **For Linux**: Reads proxy configuration from environment variables or, in case those are not
```
Comment (chasewilson) [inline reference/7.1/Microsoft.PowerShell.Utility/Invoke-RestMethod.md:1243]:
```suggestion
- **For Linux**: Reads proxy configuration from environment variables or, if those are not
```

File: reference/7.1/Microsoft.PowerShell.Utility/Invoke-WebRequest.md [hunk 7]
```diff
@@ -1189,29 +1189,27 @@ Beginning with PowerShell 6.0.0 `Invoke-WebRequest` supports basic parsing only.
 For more information, see
 [BasicHtmlWebResponseObject](/dotnet/api/microsoft.powershell.commands.basichtmlwebresponseobject).
 
-Because of changes in .NET Core 3.1, PowerShell 7.0 and higher supports proxy configuration defined
-by the following environment variables:
+Because of changes in .NET Core 3.1, PowerShell 7.0 and higher use the
+[HttpClient.DefaultProxy](/dotnet/api/system.net.http.httpclient.defaultproxy?view=netcore-3.1)
+Property to determine the proxy configuration.
```
Comment (chasewilson) [inline reference/7.1/Microsoft.PowerShell.Utility/Invoke-WebRequest.md:1194]:
```suggestion
property to determine the proxy configuration.
```

File: reference/7.1/Microsoft.PowerShell.Utility/Invoke-WebRequest.md [hunk 8]
```diff
@@ -1189,29 +1189,27 @@ Beginning with PowerShell 6.0.0 `Invoke-WebRequest` supports basic parsing only.
...
 
-- `$env:http_proxy` or `$env:HTTP_PROXY`
+The value of this property is different rules depending on your platform:
 
-  Set this variable to the endpoint of an HTTP proxy. For example: `http://127.0.0.1:3001`
+- **For Windows**: Reads proxy configuration from environment variables or, if those are not
+  defined, from the user's proxy settings.
+- **For macOS**: Reads proxy configuration from environment variables or, if those are not defined,
+  from the system's proxy settings.
+- **For Linux**: Reads proxy configuration from environment variables or, in case those are not
```
Comment (chasewilson) [inline reference/7.1/Microsoft.PowerShell.Utility/Invoke-WebRequest.md:1202]:
```suggestion
- **For Linux**: Reads proxy configuration from environment variables or, if those are not
```

File: reference/7.0/Microsoft.PowerShell.Utility/Invoke-RestMethod.md (hunk 1 above)
Comment (sdwheeler) [reply reference/7.0/Microsoft.PowerShell.Utility/Invoke-RestMethod.md]: fixed

File: reference/7.0/Microsoft.PowerShell.Utility/Invoke-RestMethod.md (hunk 2 above)
Comment (sdwheeler) [reply reference/7.0/Microsoft.PowerShell.Utility/Invoke-RestMethod.md]: fixed

File: reference/7.0/Microsoft.PowerShell.Utility/Invoke-WebRequest.md (hunk 3 above)
Comment (sdwheeler) [reply reference/7.0/Microsoft.PowerShell.Utility/Invoke-WebRequest.md:1193]: fixed

File: reference/7.0/Microsoft.PowerShell.Utility/Invoke-WebRequest.md (hunk 4 above)
Comment (sdwheeler) [reply reference/7.0/Microsoft.PowerShell.Utility/Invoke-WebRequest.md]: fixed

File: reference/7.1/Microsoft.PowerShell.Utility/Invoke-RestMethod.md (hunk 5 above)
Comment (sdwheeler) [reply reference/7.1/Microsoft.PowerShell.Utility/Invoke-RestMethod.md:1235]: fixed

File: reference/7.1/Microsoft.PowerShell.Utility/Invoke-RestMethod.md (hunk 6 above)
Comment (sdwheeler) [reply reference/7.1/Microsoft.PowerShell.Utility/Invoke-RestMethod.md:1243]: fixed

File: reference/7.1/Microsoft.PowerShell.Utility/Invoke-WebRequest.md (hunk 7 above)
Comment (sdwheeler) [reply reference/7.1/Microsoft.PowerShell.Utility/Invoke-WebRequest.md:1194]: fixed

File: reference/7.1/Microsoft.PowerShell.Utility/Invoke-WebRequest.md (hunk 8 above)
Comment (sdwheeler) [reply reference/7.1/Microsoft.PowerShell.Utility/Invoke-WebRequest.md:1202]: fixed

**Pull request merged**